Example-Nitrozen-SwiftUI/Example-Nitrozen-SwiftUI/Example-Code/EmptyView.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  EmptyView.swift
+//  Example-Nitrozen-SwiftUI
+//
+//  Created by Keyur Patel on 20/04/23.
+//
+
+import SwiftUI
+import Nitrozen_SwiftUI
+
+struct EmptyView: View {
+    var body: some View {
+        
+        List {
+            
+            Section {
```

Comment:
use navigation link and show empty view in nextScreen -- so we can have clear idea about spacing, background color and other things propertly, and in readMe-screenshot you can directly attach screenshot of last screens